README.md

```diff
@@ -222,37 +225,83 @@ $ nf-core download methylseq --singularity
 
 
 INFO: Saving methylseq
- Pipeline release: 1.0
+ Pipeline release: 1.4
  Pull singularity containers: Yes
- Output directory: nf-core-methylseq-1.0
+ Output file: nf-core-methylseq-1.4.tar.gz
 
 INFO: Downloading workflow files from GitHub
 
+INFO: Downloading centralised configs from GitHub
+
 INFO: Downloading 1 singularity container
-nf-core-methylseq-1.0.simg [762.28MB]  [####################################]  780573/780572
+
+INFO: Building singularity image from dockerhub: docker://nfcore/methylseq:1.4
+INFO:    Converting OCI blobs to SIF format
+INFO:    Starting build...
+Getting image source signatures
+....
+INFO:    Creating SIF file...
+INFO:    Build complete: /domus/h1/phil/GitHub/nf-core/tools/testing/nf-core-methylseq-1.4/singularity-images/nf-core-methylseq-1.4.simg
```

Comment:
Do you want to leave this path?

Reply:
Yeah, I guess it would look a bit nicer if it was generalised.. 👍